.idea/jsLibraryMappings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.idea files don't really need to be committed since they're specific to the user's environment